ext/json/json.c

```diff
@@ -95,6 +96,9 @@ static PHP_MINIT_FUNCTION(json)
 	INIT_CLASS_ENTRY(ce, "JsonSerializable", json_serializable_interface);
 	php_json_serializable_ce = zend_register_internal_interface(&ce);
 
+	INIT_CLASS_ENTRY(ce, "JsonException", NULL);
```

Comment:
Technically I'm not sure if we should introduce `JsonException` without RFC. It really goes against the linked class-naming RFC and if it's `JSONException` then it goes against consistency of this extension. I think we need an RFC in any case.